src/natcap/invest/urban_nature_access.py

```diff
@@ -97,19 +100,34 @@
                 "pixel."
             ),
         },
-        'admin_unit_vector_path': {
+        'aoi_vector_path': {
             'type': 'vector',
             'name': 'administrative boundaries',
             'geometries': spec_utils.POLYGONS,
-            'fields': {},  # TODO, complete required fields (if any)
-            'about': "",  # TODO, will know more about this when I implement.
+            'fields': {
+                "pop_[POP_GROUP]": {
+                    "type": "number",
```

Comment:
To be more specific, this could be `"ratio"` instead of `"number"`, and then we could remove the explanation `(numeric values in the range 0-1)` in the about text.

Reply:
Makes sense!  Patched in 0ff3baffd